prisma/schema.prisma

```diff
@@ -424,6 +424,9 @@ model Event {
   // metadata stored in the event
   additionalMetadata Json?
 
+  // this is used to order the events on bulk inserts
+  insertOrder Int?
```

Comment:
note: not for this PR, but putting it into the universe. 
i think we may want to set something like an int unix time sdk side to expose more implicit order control for our customers so this is a step in the right direction